tests/src/test/scala/actionContainers/DotNet3_1ActionContainerTests.scala

```diff
@@ -100,6 +101,19 @@ class DotNet3_1ActionContainerTests extends BasicActionRunnerTests with WskActor
     })
   }
 
+  it should "support a large payload" in {
+    val (out, err) = withActionContainer() { c =>
+      val sb = new StringBuilder(18000000 + functionb64.length());
```

Comment:
```suggestion
      val sb = " " * (18000000 + functionb64.length())
```

Reply:
Thanks for your suggestion, ive committed your suggestion with a small change in 1600d77